static/elements/chromedash-approvals-dialog.js

```diff
@@ -379,26 +392,92 @@ class ChromedashApprovalsDialog extends LitElement {
       this.renderApproval(apprDef));
   }
 
-  renderComment(comment) {
+  // Returns a boolean representing whether the given comment can be edited.
+  commentIsEditable(comment) {
+    // If the comment has been deleted, it should only be editable
+    // by site admins or the user who deleted it.
+    if (comment.deleted_by) {
+      return this.user.is_admin || comment.deleted_by === this.user.email;
+    }
+    // If the comment is not deleted, site admins or the author can edit.
+    return this.user.email === comment.author || this.user.is_admin;
+  }
+
+  // Format a message to show on a deleted comment the user can edit.
+  formatCommentDeletedPreface(comment) {
```

Comment:
I think that the conventional verb to use in these method names would be 'render'.  So, this would be `renderCommentDeletedPreface`.

Reply:
Name changed